### ISSUE TITLE
Confusion in performance

### ISSUE THREAD

**Issue by linqinghong** [speedinghzl/pytorch-segmentation-toolbox#31]
hi,thanks for you nice work!
I want to ask same question:
In your implementation,what is your performance in cityscapes validation set and in what condition?
I meet some confusion because i use your project only get 77.51 which batchsize is 12 in 4GPU and 77.40 which batchsize is 24 in 8GPU in val.but in the paper the mIoU can reach 78.5%.
have you reach this result?
thanks!

**Comment by speedinghzl** [comment]
The settings are listed in [run_local.sh](https://github.com/speedinghzl/pytorch-segmentation-toolbox/blob/master/run_local.sh). The batch size is 8 on 4 GPUs. I think the settings are clear and easy to follow.

**Comment by linqinghong** [comment]
I have seen the run_local.sh, but i want to make sure that have you train model in pytorch 0.4 and close the ohem.thank!

**Comment by speedinghzl** [comment]
The answer is YES.
